README.md

```diff
@@ -1,5 +1,7 @@
 # SNEWPY: Supernova Neutrino Early Warning Models for Python
 
+![SNEWPY logo](/doc/source/snewpy-logo.png)
```

Comment:
The logo displayed in the README is rather large, it makes me scroll down to get to the text. 
Maybe we should scale it down:
```suggestion
<img alt="SNEWPY" src="/doc/source/snewpy-logo.png" height=100px/>
```

Reply:
Good point, thanks! Added this now; I also set a max-width for very narrow screens/windows.